Agents/utils/GeoServerJwtProxy/stack-manager-input/geoserver-jwt-proxy.json

```diff
@@ -0,0 +1,41 @@
+{
+    "ServiceSpec": {
+        "Name": "geoserver-jwt-proxy",
+        "TaskTemplate": {
+            "ContainerSpec": {
+                "Image": "ghcr.io/cambridge-cares/geoserver-jwt-proxy:1.0-SNAPSHOT",
+                "Env": [
+                    "VIEWPARAM_NAME=device_id"
```

Comment:
this one is it better to make it user_id so it is same as the current geoserver setting?

Reply:
This an example, there is another one in the timeline stack that uses user_id